pjnath/include/pjnath/config.h

```diff
@@ -286,7 +286,7 @@
 /**
  * Maximum number of ICE components.
  */
-#define PJ_ICE_MAX_COMP				    (2<<PJ_ICE_COMP_BITS)
+#define PJ_ICE_MAX_COMP				    (1<<PJ_ICE_COMP_BITS)
```

Comment:
@nanangizz question. What motivated this change?

Reply:
Shouldn't it the correct formula? E.g: if `COMP_BITS` is 1, it can only represent maximum of 2 components, the prev formula will result in 4 components.

Anything becomes not working or having a problem after that change? :)

Reply:
Yeah you are right. In our case it COMP_BITS should be 4

For the little story, I started to rebase our code on pjsip 2.11 (we're using 2.10) and got some assertions failure when the call starts with
https://review.jami.net/c/ring-daemon/+/18710/3/contrib/src/pjproject/0007-upnp-srflx-nat-assisted-cand.patch

But our ICE uses 4 components for medias, so COMP_BITS should be set to 2